src/helix-ui/styles/modules/typography.less

```diff
@@ -1,90 +0,0 @@
-// RESET
-h1, h2, h3, h4, h5, h6 {
-  font-family: @font-sans-serif;
-  font-weight: 400;
-  line-height: 1.5;
-  margin: 0;
-
-  & + * {
-    margin-top: 0.5rem;
-  }
-}
```

Comment:
* above moved to `base/typography.less`
* remainder moved to `components/typography.less`